extras/systemd/dynflow-sidekiq@.service

```diff
@@ -4,13 +4,23 @@ Documentation=https://theforeman.org
 After=network.target remote-fs.target nss-lookup.target
 
 [Service]
-Type=forking
+Type=simple
 User=foreman
 TimeoutSec=90
 Environment=RAILS_ENV=production
 Environment=DYNFLOW_SIDEKIQ_SCRIPT=/usr/share/foreman/extras/dynflow-sidekiq.rb
+# Greatly reduce Ruby memory fragmentation and heap usage
+# https://www.mikeperham.com/2018/04/25/taming-rails-memory-bloat/
+Environment=MALLOC_ARENA_MAX=2
 WorkingDirectory=/usr/share/foreman
 ExecStart=/usr/bin/sidekiq -e ${RAILS_ENV} -r ${DYNFLOW_SIDEKIQ_SCRIPT} -C /etc/foreman/dynflow/%i.yml
+ExecReload=/usr/bin/kill -TSTP $MAINPID
+
+SyslogIdentifier=dynflow-sidekiq@%i
```

Comment:
Probably good to also use `%i` in `Description` (line 2).

Reply:
Thanks :+1: added